modules/authentication/src/Authentication.ts

```diff
@@ -57,9 +58,11 @@ export default class Authentication extends ManagedModule<Config> {
   async onServerStart() {
     await this.grpcSdk.waitForExistence('database');
     this.database = this.grpcSdk.database!;
-    TokenProvider.getInstance(this.grpcSdk);
     await this.registerSchemas();
-    await runMigrations(this.grpcSdk);
+    const version = ManifestManager.getInstance().moduleVersion;
```

Comment:
can we add the registerMigrations logic to module manager? That would be cleaner I think, and the modules can optionally specify a migrationsPath variable so that they can modify migration location.